src/content/docs/en/reference/experimental-flags/svg-optimization.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Experimental SVG optimization
+sidebar:
+  label: SVG optimization
+i18nReady: true
+---
+
+import Since from '~/components/Since.astro'
+
+<p>
+
+**Type:** `object`<br />
+**Default:** `undefined`<br />
+<Since v="5.8.0" />
+</p>
+
+This experimental feature enables automatic optimization of SVG assets using [SVGO](https://svgo.dev/) during build time.
+
+When enabled, all imported SVG files will be optimized for smaller file sizes and better performance while maintaining visual quality. This can significantly reduce the size of your SVG assets by removing unnecessary metadata, comments, and redundant code.
+
+To enable this feature, add the experimental flag in your Astro config:
+
+```js title="astro.config.mjs" ins={4-8}
+import { defineConfig } from "astro/config"
+
+export default defineConfig({
+  experimental: {
+    svg: {
+      optimize: true
+    }
+  }
+})
+```
+
+## Configuration
+
+### `optimize`
+
+**Type:** `boolean`<br />
+**Default:** `true`
+
+Whether to enable SVG optimization using SVGO during build time.
+
+When enabled, all imported SVG files will be optimized for smaller file sizes and better performance while maintaining visual quality.
+
+```js title="astro.config.mjs"
+export default defineConfig({
+  experimental: {
+    svg: {
+      optimize: true
+    }
+  }
+})
+```
+
+### `svgoConfig`
+
+**Type:** `Config` (SVGO configuration object)<br />
+**Default:** `{}`
+
+Configuration object passed directly to SVGO for customizing SVG optimization.
+
+See [SVGO documentation](https://svgo.dev/docs/preset-default/) for available options and plugins.
```

Comment:
This link only appears to go to information about the `preset-default` and not plugins in general, nor other available options. (I'm wondering if https://svgo.dev/docs/plugins/ is maybe a better link to lead people to for plugins info, but also, below you show using `floatPrecision` outside of the plugins array and it's not obvious where that comes from, or what else is available to use. (Is that how you pass paramaters to a plugin? In the SVGO documentation, it appears that `floatPrecision` is a parameter of a plugin, and not a separate thing itself)